tests/Pest.php

```diff
@@ -28,6 +33,6 @@
 |
 */
 
-expect()->extend('toBeOne', function () {
-    return $this->toBe(1);
-});
+// expect()->extend('toBeOne', function () {
+//     return $this->toBe(1);
+// });
```

Comment:
Can this all be removed?

Reply:
I left it just as an example for how we might write a custom assertion. I suppose we could remove it, but I don't think it is an issue.